src/pool/srv_pool.c

```diff
@@ -1162,6 +1186,15 @@ pool_svc_locate_cb(d_iov_t *id, char **path)
 	return 0;
 }
 
+static unsigned int
+get_crt_event_delay(void)
+{
+	unsigned int t = 0 /* s */;
+
+	d_getenv_uint("CRT_EVENT_DELAY", &t);
```

Comment:
Yes, the feature is currently disabled by default, mostly because I don't know how to modify the test framework yet. (All previous CI runs of this PR were using 10 s delays.) May be changed in a follow-on PR.

Is there any strong reason for introducing a maximum-allowed value? If we have a particular max value in mind, I'm happy to consider it.

Reply:
can add "CRT_EVENT_DELAY=0" to src/tests/ftest/util/server_utils_params.py, similar as "DAOS_POOL_RF=4", just FYI.

"Is there any strong reason for introducing a maximum-allowed value?" just avoid admin set it to a unreasonable large number. looks fine to leave it as now if we don't know which max number is reasonable.

Reply:
Thanks for the helpful pointer. I've changed the default to 10 s, and set this delay to 1 s for CI runs, to get some coverage while adding minimum extra delay. Hopefully that will work.